crates/router/src/connector/payme/transformers.rs

```diff
@@ -997,17 +992,25 @@ impl TryFrom<types::RefundsResponseRouterData<api::Execute, PaymeRefundResponse>
         let refund_status = enums::RefundStatus::try_from(item.response.sale_status.clone())?;
         let response = if is_refund_failure(refund_status) {
             let payme_response = &item.response;
+            let status_error_code = payme_response.status_error_code.ok_or(
```

Comment:
Don't validate the field and throw error. Instead if the value is None default error_code, error_message to NO_ERROR_CODE & NO_ERROR_MESSAGE